client/my-sites/importer/importing-pane.jsx

```diff
@@ -117,7 +117,7 @@ class ImportingPane extends React.PureComponent {
 
 	getHeadingText = () => {
 		return translate(
-			'Importing takes 15 minutes or a while longer if your site has a lot of media. ' +
+			'Importing may take more than 15 minutes if your site has a lot of media. ' +
```

Comment:
What do you think about avoiding mentioning 15 minutes altogether and reverting to the [old version](https://github.com/Automattic/wp-calypso/blame/9f8620b62390e3730437103b8ab961bbff6e2dca/client/my-sites/importer/importing-pane.jsx#L116), something like:
`Importing may take a while, but you can safely navigate away from this page if you need to: we'll send you a notification when it's done.`

Reply:
Good suggestion. I adapted it a bit to be closer to the current message.